arm/Microsoft.RecoveryServices/vaults/protectionContainers/readme.md

```diff
@@ -36,6 +37,10 @@ This module deploys a Protection Container for a Recovery Services Vault
 | `sourceResourceId` | string | `''` |  | Resource ID of the target resource for the Protection Container. |
 
 
+### Parameter Usage: `<ParameterPlaceholder>`
+
+// TODO: Fill in Parameter usage
+
```

Comment:
Please either add parameter usage if needed or remove this placeholder

Reply:
Done

Reply:
Could you kindly double check? I still see the old code

Reply:
Not sure, I can see the correct code in my branch